src/SerializerFactory.php

```diff
@@ -94,15 +111,25 @@ private function shouldSerializeReferenceSnaksWithHash() {
 	}
 
 	/**
-	 * Returns a Serializer that can serialize Item and Property objects.
-	 *
-	 * @return Serializer
+	 * @throws IllegalValueException
+	 * @return Serializer A dispatching serializer that can serialize whatever the entity
```

Comment:
Shouldn't this then say `@return DispatchingSerializer`? Or is it only for compliance with the interface?